package.json

```diff
@@ -25,7 +25,9 @@
   "repository": "git://github.com/bpanel-org/price.git",
   "dependencies": {
     "babel-cli": "^6.26.0",
-    "babel-plugin-transform-runtime": "^6.23.0"
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "bcurl": "pinheadmz/bcurl#bpanel-dev",
+    "@bpanel/curl": "^0.0.1"
```

Comment:
just fyi this will fail cause npm install to fail until it's published. I'm also wondering if it's better to add this to peerDependencies for now just because installing it isn't enough, it also needs to be in the config file so the server knows to import it.